var/spack/repos/builtin/packages/redis/package.py

```diff
@@ -33,3 +33,8 @@ def install_targets(self):
             'PREFIX={0}'.format(self.spec.prefix),
             'install'
         ]
+
+    def install(self, spec, prefix):
+        make(*self.install_targets)
+        mkdirp(prefix.conf)
+        install('redis.conf', join_path(prefix.conf, 'redis.conf'))
```

Comment:
```suggestion
    @run_after('install')
    def install_conf(self):
        mkdirp(self.prefix.conf)
        install('redis.conf', self.prefix.conf)
```
This is how I would do things.